packages/core/data-transfer/src/engine/__tests__/engine.test.ts

```diff
@@ -1116,7 +1116,7 @@ describe('Transfer engine', () => {
 
       // Source that detects when it is paused
       const source = {
-        ...createSource({ entities: sourceData }),
+        ...createSource({ entities: sourceData as any }),
```

Comment:
Why do we need this?

Reply:
test data wasn't implemented fully, I just updated it